### PR TITLE
feature/EVS-NWPS_plots

### PR DESCRIPTION
**Pull request by SamiraArdani-NOAA**
<b>Note to developers: You must use this PR template!</b>

## Description of Changes

> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.

This PR initializes the plots step for EVS-NWPS.

## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by?
No.
* Do you have any planned upcoming annual leave/PTO?
No.
* Are there any changes needed for when the jobs are supposed to run?
No.
- [x] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [x] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [x] References the feature branch for `HOMEevs` are removed from the code.
- [x] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [x] Jobs over 15 minutes in runtime have restart capability.
- [x] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [x] Jobs contain the approriate file checking and don't run METplus for any missing data.
- [x] Code is using METplus wrappers structure and not calling MET executables directly.
- [ ] Log is free of any ERRORs or WARNINGs.
There is a possibility of WARNING messages due to missing data.

## Testing Instructions

> Please include testing instructions for the PR assignee. Include all relevant input datasets needed to run the tests.
1- To test this PR, you need to run the driver script for only plots step.
2- Clone my fork in your local and checkout branch feature/EVS-NWPS_plots.
3- ln -s fix directory.
4- In COMIN, replace the ${USER} with emc.vpppg.
5- Run the driver scripts for plots step located at: EVS/dev/drivers/scripts/plots/nwps/*.

### CODE REVIEW

File: dev/drivers/scripts/plots/nwps/jevs_nwps_wave_grid2obs_plots.sh [hunk 1]
```diff
@@ -12,8 +12,9 @@ set -x
 export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
 
 export MODELNAME=nwps
-export OBTYPE=NDBC
-export NET=evs
+export OBTYPE=NDBC_STANDARD
+export MASK=GOM
+export NET=evs_beta5
```
Comment (malloryprow) [inline dev/drivers/scripts/plots/nwps/jevs_nwps_wave_grid2obs_plots.sh]: Change NET to evs (`NET=evs`)

**Comment by malloryprow** [comment]
Should I wait an hour to turn the plots at 1530Z?

**Comment by SamiraArdani-NOAA** [comment]
> Should I wait an hour to turn the plots at 1530Z?

@malloryprow, Ok. Sounds good to me.

**Comment by malloryprow** [comment]
### jevs_nwps_wave_grid2obs_plots - Try 1
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/EVS/dev/drivers/scripts/plots/nwps/jevs_nwps_wave_grid2obs_plots.o159291254
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/evs/v2.0
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_nwps_wave_grid2obs_plots.159291254.cbqs01

**Comment by SamiraArdani-NOAA** [comment]
@malloryprow, There was no ERROR in .o file. The COMOUT includes last31days and last90days plots. I copied the files from your test to my own directory and untarred them. The plot job successfully generated time_series and lead_average plots.

**Comment by malloryprow** [comment]
Two things I noticed:

First, 
> + 9 + /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/EVS/ush/nwps/evs_wave_leadaverages.sh
> mkdir: cannot create directory â/lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_nwps_wave_grid2obs_plots.159291254.cbqs01/sfcshpâ: File exists

A check for this directory before making it should get rid of this.

Second is we can probably adjust the resources. It is requesting 128 cpus (`ncpus=128` in the driver script), but it looks like only 36 of those are actually being used (`mpiexec -np 36 `). The memory looks pretty high too. It is 500GB but looks like about ~5GB is being used (`update_job_usage: Memory usage: mem=4542056kb`). I think if it is changed to `ncpus=36:mem=50G` we should be good. I'm giving some cushion on the memory since we don't have the full 90 days of stats and that will probably bump it up some!

**Comment by SamiraArdani-NOAA** [comment]
> Two things I noticed:
> 
> First,
> 
> > * 9 + /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/EVS/ush/nwps/evs_wave_leadaverages.sh
> >   mkdir: cannot create directory â/lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_nwps_wave_grid2obs_plots.159291254.cbqs01/sfcshpâ: File exists
> 
> A check for this directory before making it should get rid of this.
> 
> Second is we can probably adjust the resources. It is requesting 128 cpus (`ncpus=128` in the driver script), but it looks like only 36 of those are actually being used (`mpiexec -np 36 `). The memory looks pretty high too. It is 500GB but looks like about ~5GB is being used (`update_job_usage: Memory usage: mem=4542056kb`). I think if it is changed to `ncpus=36:mem=50G` we should be good. I'm giving some cushion on the memory since we don't have the full 90 days of stats and that will probably bump it up some!

@malloryprow, Thanks! I did not notice that. I removed the lines that makes sfcshp directory. Those directories are redundant. The memory was adjusted as well.

File: scripts/plots/nwps/exevs_nwps_wave_grid2obs_plots.sh
Comment (malloryprow) [inline scripts/plots/nwps/exevs_nwps_wave_grid2obs_plots.sh:1]: In this script there is `mpiexec -np 36 --cpu-bind verbose,core --depth=3 cfp plot_all_${MODELNAME}_${RUN}_g2o_plots.sh`. Can we remove `--depth=3`. In a previous PR with Binbin, we found this depth setting wasn't really doing anything and everything ran fine without it.

**Comment by malloryprow** [comment]
### jevs_nwps_wave_grid2obs_plots - Try 2
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/EVS/dev/drivers/scripts/plots/nwps/jevs_nwps_wave_grid2obs_plots.o159301894
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr595/evs/v2.0
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_nwps_wave_grid2obs_plots.159301894.cbqs01

**Comment by SamiraArdani-NOAA** [comment]
@malloryprow, Everything looks good.

**Review by malloryprow — APPROVED** [review]
Changes are good and testing successful.

**Review by AndrewBenjamin-NOAA — APPROVED** [review]
I have reviewed the code and approved these changes.

**Comment by malloryprow** [comment]
Thanks @SamiraArdani-NOAA! Not sure if this was a checklist item on the Fixes and Additions document but if it does be sure to check it off!

**Pull request merged**